packages/high-roller-bot/src/bot.ts

```diff
@@ -26,7 +26,7 @@ function respond(node: Node, nodeAddress: Address, msg: UpdateStateMessage) {
     console.log(newState);
   }
 
-  if (newState.stage === 2) {
+  if (newState["stage"] === 2) {
```

Comment:
I think you could introduce a type for this inside the scope of the `high-roller-bot` package and type `newState` as that; `HighRollerAppState`.